client/components/buttons/UpvoteButton.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import handleUpvote from '../../actions/requestHandlers/handleUpvote';
+import './buttons.scss';
+
+
+const propTypes = {
+  recipe: PropTypes.objectOf(PropTypes.any).isRequired,
+  handleUpvote: PropTypes.func.isRequired,
+};
+
+class UpvoteButton extends Component {
```

Comment:
This should be rewritten to be a stateless functional component seeing as it maintains no state of it's own nor does it use any of the lifecycle methods